engine/apps/oss_installation/models/cloud_connector.py

```diff
@@ -61,7 +60,7 @@ def sync_users_with_cloud(self) -> tuple[bool, str]:
             logger.warning("Unable to sync with cloud. GRAFANA_CLOUD_ONCALL_TOKEN is not set")
             error_msg = "GRAFANA_CLOUD_ONCALL_TOKEN is not set"
 
-        existing_emails = list(User.objects.filter(role__in=(Role.ADMIN, Role.EDITOR)).values_list("email", flat=True))
+        existing_emails = list(User.objects.filter().values_list("email", flat=True))
```

Comment:
halpp

Reply:
I'm a bit unsure what permission(s) that should be used here. Once I figure out the permission(s) the query would look as such:
```python
from apps.api.permissions import GrafanaPermission # GrafanaPermission is a typing.TypedDict

existing_emails = list(User.objects.filter(permissions__contains=GrafanaPermission(
  action="<permission-string>")).values_list("email", flat=True))
```